### PR TITLE
[v6r13] Pass cached request as json string to the ReqManager

### PR DESCRIPTION
**Pull request by atsareg**

### CODE REVIEW

File: RequestManagementSystem/Service/ReqProxyHandler.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def sweeper( cls ):
           requestString = "".join( open( cachedFile, "r" ).readlines() )
           cachedRequest = eval( requestString )
```
Comment (sposs) [inline RequestManagementSystem/Service/ReqProxyHandler.py]: This hurts my eyes a lot... Can someone put cjson.decode instead?

Comment (chaen) [inline RequestManagementSystem/Service/ReqProxyHandler.py]: Hum, I just realized now... I think that I did not adapt the ReqProxyHandler to the new json system. So not sure that this fix will make it work

Comment (atsareg) [inline RequestManagementSystem/Service/ReqProxyHandler.py]: Please, do it properly please.

**Comment by chaen** [comment]
Andrei, can you please cancel that one, I will make one which takes into account all the changes implied by the json module.

**Comment by chaen** [comment]
and you really want it against integration ?

**Comment by atsareg** [comment]
Premature fix

**Pull request closed without merging**